src/helpers/agGrid.ts

```diff
@@ -260,6 +263,60 @@ export const getRow = (
     return row;
 };
 
+export const getRowTotals = (
```

Comment:
I think this function deserves some unit test

Reply:
done